### DISCUSSION TITLE
nvptx64-nvidia-cuda-sycldevice question

### DISCUSSION THREAD

**Discussion by zjin-lcf** [intel/llvm#4654] [General]
There is a question about "-fsycl-targets=nvptx64-nvidia-cuda-sycldevice"

"-sycldevice" is not mentioned in user manual, is it ?  Do you recommend users to remove "-sycldevice" in a Makefile ?

Thanks

**Comment by alexbatashev** [comment]
We have removed that prefix ~3 weeks ago. See: https://github.com/intel/llvm/pull/3929 and https://github.com/intel/llvm/pull/4567
It has been deprecated and users are encouraged to use triples without that postfix.
